### PR TITLE
Chore: Updates FastStore Contributing Guide

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

Updates contributing guide to reduce friction for contributors.

## How it works?

- Updates the contributing file (added more details and instructions about the process and label)
- Updates the pull request template

## How to test it?

Check the modified files and review it.
[Preview](https://github.com/vtex/faststore/blob/4adeb901b5bc031161111f0078d3fcd2b8716efb/CONTRIBUTING.MD)

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
+Choose a branch name that reflects the work being done, such as `fix/query-typo` or `feat/add-new-component`.
+
+### Testing Changes on a Store
 
 To test your changes in a store, you will need to create a pull request (for more guidance, refer to the next section). You can keep your pull request in `draft` while testing.
 
-1. After committing your changes, push them to the remote repo and open a Pull Request.
+1. After committing your changes, push them to the remote repository and open a Pull Request.
 
 2. In the checks section, find `ci/codesandbox` and click on `Details`.
```
Comment (eduardoformiga) [inline CONTRIBUTING.MD:62]: non-blocking: maybe we could change the image below to use faststore/cli instead of the core?

**Review by eduardoformiga — APPROVED** [review]
LGTM! ⭐

File: CONTRIBUTING.MD (hunk 1 above)
Comment (hellofanny) [reply CONTRIBUTING.MD:62]: Suree! I was about to update it, thanks for the reminder 🌟

Comment (hellofanny) [reply CONTRIBUTING.MD:62]: Updated. [2e196d5](https://github.com/vtex/faststore/pull/2476/commits/2e196d557f71a9f2233370237ffd89232234e792).

**Review by renatamottam — APPROVED** [review]
Nice one, fan!

**Pull request merged**